frontend/packages/client/src/App.sass

```diff
@@ -15,7 +15,7 @@ $family-monospace: "DM Sans"
 $body-color: black
 $body-background-color: white
 $primary: $yellow
-$spacing-values: ("0": 0, "1": 0.25rem, "2": 0.5rem, "3": 0.75rem, "4": 1rem, "5": 1.5rem, "6": 3rem, "7": 4.5rem,"8": 6rem, "9": 9rem, "auto": auto) !default
+$spacing-values: ("0": 0, "1": 0.25rem, "2": 0.5rem, "3": 0.75rem, "4": 1rem, "5": 1.5rem, "5_5": 2.5rem, "6": 3rem, "7": 4.5rem,"8": 6rem, "9": 9rem, "auto": auto) !default
 
```

Comment:
This is not the most happy way of doing but it looks like there's a lot of components that require padding of 40px